docker-compose.dev.yaml

```diff
@@ -139,6 +330,109 @@ services:
       - VLLM_ATTENTION_BACKEND=FLASH_ATTN
+      - VLLM_FLASH_ATTN_VERSION=3
+      - VLLM_USE_V1=0
+      - CUDA_VISIBLE_DEVICES=0
+    ipc: host
+    command: ${VLLM_ENGINE_ARGS}
+
+  vllm4:
+    <<: *inference-service-cuda
+    container_name: chat-completions4
+    profiles: [chat_completions_vllm]
+    image: vllm/vllm-openai:v0.8.1
+    environment:
+      # Backend for attention computation
+      # Available options:
+      # - "TORCH_SDPA": use torch.nn.MultiheadAttention
+      # - "FLASH_ATTN": use FlashAttention
+      # - "XFORMERS": use XFormers
+      # - "ROCM_FLASH": use ROCmFlashAttention
+      # - "FLASHINFER": use flashinfer (recommended for fp8 quantized models)
+      - VLLM_ATTENTION_BACKEND=FLASH_ATTN
+      - VLLM_FLASH_ATTN_VERSION=3
+      - VLLM_USE_V1=0
+      - CUDA_VISIBLE_DEVICES=0
+    ipc: host
+    command: ${VLLM_ENGINE_ARGS}
+
+  vllm5:
+    <<: *inference-service-cuda
+    container_name: chat-completions5
+    profiles: [chat_completions_vllm]
+    image: vllm/vllm-openai:v0.8.1
+    environment:
+      # Backend for attention computation
+      # Available options:
+      # - "TORCH_SDPA": use torch.nn.MultiheadAttention
+      # - "FLASH_ATTN": use FlashAttention
+      # - "XFORMERS": use XFormers
+      # - "ROCM_FLASH": use ROCmFlashAttention
+      # - "FLASHINFER": use flashinfer (recommended for fp8 quantized models)
+      - VLLM_ATTENTION_BACKEND=FLASH_ATTN
+      - VLLM_FLASH_ATTN_VERSION=3
+      - VLLM_USE_V1=0
+      - CUDA_VISIBLE_DEVICES=0
+    ipc: host
+    command: ${VLLM_ENGINE_ARGS}
+
+  vllm6:
+    <<: *inference-service-cuda
+    container_name: chat-completions6
+    profiles: [chat_completions_vllm]
+    image: vllm/vllm-openai:v0.8.1
+    environment:
+      # Backend for attention computation
+      # Available options:
+      # - "TORCH_SDPA": use torch.nn.MultiheadAttention
+      # - "FLASH_ATTN": use FlashAttention
+      # - "XFORMERS": use XFormers
+      # - "ROCM_FLASH": use ROCmFlashAttention
+      # - "FLASHINFER": use flashinfer (recommended for fp8 quantized models)
+      - VLLM_ATTENTION_BACKEND=FLASH_ATTN
+      - VLLM_FLASH_ATTN_VERSION=3
+      - VLLM_USE_V1=0
+      - CUDA_VISIBLE_DEVICES=0
+    ipc: host
+    command: ${VLLM_ENGINE_ARGS}
+
+  vllm7:
+    <<: *inference-service-cuda
+    container_name: chat-completions7
+    profiles: [chat_completions_vllm]
+    image: vllm/vllm-openai:v0.8.1
+    environment:
+      # Backend for attention computation
+      # Available options:
+      # - "TORCH_SDPA": use torch.nn.MultiheadAttention
+      # - "FLASH_ATTN": use FlashAttention
+      # - "XFORMERS": use XFormers
+      # - "ROCM_FLASH": use ROCmFlashAttention
+      # - "FLASHINFER": use flashinfer (recommended for fp8 quantized models)
+      - VLLM_ATTENTION_BACKEND=FLASH_ATTN
+      - VLLM_FLASH_ATTN_VERSION=3
+      - VLLM_USE_V1=0
+      - CUDA_VISIBLE_DEVICES=0
+    ipc: host
+    command: ${VLLM_ENGINE_ARGS}
+
+  vllm8:
```

Comment:
Do you need 8 different running vllm instances ?

Reply:
Ideally at some point yes but for now it's not required. I will leave it though since it doesn't negatively impact the dev environment